{{cookiecutter.project_slug}}/docs/index.md

```diff
@@ -0,0 +1 @@
+--8<-- "README.md"
```

Comment:
In the past I've done
```md
{%
   include-markdown "../README.md"
%}
```
using the `mkdocs-include-markdown-plugin`

Reply:
I have no strong views on `pymdownx.snippets` Markdown extension versus this plug-in. Syntax for the `mkdocs-include-markdown-plugin` does seem clearer which is nice. Not sure if there is any advantage of something which is included specifically as a `markdown_extension` versus a `plugin`?

Reply:
I have no idea to be honest. I haven't used the other one, the syntax is very weird though!

Reply:
In case you didn't get it, as I didn't until reading the docs, the `8<` is meant to me an ASCII set of ✂️, but agree it looks weird. 
```
{%
   include-markdown "../README.md"
%}
```
is much more explicit (I also didn't like the fact the paths for snippets extension are relative to `mkdocs.yml` rather than the actual Markdown file they are included in) so will double check this works fine here and if so switch

Reply:
Sounds good. And thanks for the explanation!